PWGGA/GammaConvBase/AliConversionPhotonCuts.cxx

```diff
@@ -721,93 +723,93 @@ Bool_t AliConversionPhotonCuts::InitPIDResponse(){
   return kFALSE;
 }
 ///________________________________________________________________________
-Bool_t AliConversionPhotonCuts::LoadElecDeDxPostCalibration(Int_t runNumber) {
+Bool_t AliConversionPhotonCuts::InitializeElecDeDxPostCalibration(TString filename) {
+  AliInfo("Entering loading of correction map for post calibration");
 
-  if(runNumber==fRecalibCurrentRun)
-    return kTRUE;
-  else
-    fRecalibCurrentRun=runNumber;
+  TFile *file = TFile::Open(filename.Data());
+  if(!file){
+    AliError(Form("file for electron dEdx post calibration %s not found",filename.Data()));
+    return kFALSE;
+  }else{
+    AliInfo(Form("found %s ",filename.Data()));
+  }
 
-  if (fFileNameElecDeDxPostCalibration!="")
-  { //if fFileNameElecDeDxPostCalibration specified in the AddTask via SetElecDeDxPostCalibrationCustomFile()
-    AliInfo(Form("Loading dEdx recalibration maps from given path %s",fFileNameElecDeDxPostCalibration.Data()));
+  for(Int_t i=0;i<fnRBins;i++){
+   if (fIsRecalibDepTPCCl){
+    fHistoEleMapRecalib[i]  = (TH2S*)file->FindObject(Form("Ele_Cl%d_recalib",i));
```

Comment:
Please also use Get() instead of FindObject() here, otherwise it might now function properly.